### ISSUE TITLE
Custom color scheme not working in Kubuntu 18.04

### ISSUE THREAD

**Issue by BrianD91** [dpayne/cli-visualizer#96]
I just installed vis and I tried setting up my own custom color scheme. 

I have the color scheme in the correct directory, its a few lines of different hex color values to make a gradient, and I have that color scheme selected in the config file, but my changes don't seem to be applying properly. I'm using Konsole as my terminal window if that makes a difference

**Comment by Mr-Turtles** [comment]
Same here. I did some messing around with it and it seems like the issue is that we are unable to switch from scheme to scheme and we're stuck on the default one and can't even get into the prepackaged "rainbow" scheme.

**Comment by crozbo** [comment]
Same on arch linux with KDE Plasma and pulse audio. Config file ~/.config/vis/config or starting vis vith -c  config dont have any effect. All configs stay on defaults.

**Comment by ViperZer0** [comment]
https://pastebin.com/pw3YrkwH

It looks like the visualizer isn't even attempting to load the custom colors. Hope that helps!

**Comment by dpayne** [comment]
Yeah I think you're right. Could you also post your config and the output of `ls -altrR ~/.config/vis/`
